common/static/coffee/src/discussion/views/discussion_thread_view.coffee

```diff
@@ -118,6 +118,11 @@ if Backbone?
         @responsesRequest.abort()
 
     loadResponses: (responseLimit, elem, firstLoad) ->
+      # takeFocus take the page focus to response loading element while responses are being fetched.
+      # In tab-mode, responses are loaded automatically, Do not scroll to the element(TNL-1530)
+      # In inline-mode, user clicks 'expand' to open responses, Its ok to scroll to the element (Default)
+      takeFocus = true
+      takeFocus = false if @mode == "tab"
```

Comment:
`if @mode == "tab" then false else true` would work too, right?

"When viewing in the Discussions tab" and "When viewing inline in courseware" would make the second and third lines of the comment a bit clearer.